docs/developer-guide/syntax-reference.md

```diff
@@ -99,6 +99,31 @@ You can create your own aggregation logic by implementing a User Defined
 Aggregation Function (UDAF). For more information, see
 [UDAFs](../concepts/functions.md#udafs).
 
+Aggregations return results per partition. To get results across all partitions,
+you can use PARTITION BY with a fixed key, for example:
```

Comment:
Maybe this whole example with `PARTITION BY` is no longer needed? Instead, could say

> To get results across all partitions, you can use `GROUP BY` with a fixed key

and just show the `group by` example below. Probably best to get input from the ksqlDB team on whether that's reasonable or not.